indra/newview/llviewerregion.cpp

```diff
@@ -2499,10 +2499,15 @@ void LLViewerRegion::setSimulatorFeatures(const LLSD& sim_features)
         };
 
     auto workqueue = LL::WorkQueue::getInstance("mainloop");
+
     if (workqueue)
     {
         LL::WorkQueue::postMaybe(workqueue, work);
```

Comment:
`postMaybe()` restates the `if` on line 2503, so you could just call `post()` here.